src/ngraph/runtime/executable.hpp

```diff
@@ -44,6 +44,16 @@ class NGRAPH_API ngraph::runtime::Executable
     virtual bool call(const std::vector<std::shared_ptr<runtime::Tensor>>& outputs,
                       const std::vector<std::shared_ptr<runtime::Tensor>>& inputs) = 0;
 
+    /// \param outputs vector of runtime::Tensor used as outputs. Null elements will be replaced
```

Comment:
Why do we need a new call method? What is different than the other method? Why would I call this? Please document in the code